### PR TITLE
tikv: updated description of region size

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Updated description of region size.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-storage.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ To make it easy to understand, let's assume that all data only has one replica.
 * Hash: Create Hash by Key and select the corresponding storage node according to the Hash value.
 * Range: Divide ranges by Key, where a segment of serial Key is stored on a node.
 
-TiKV chooses the second solution that divides the whole Key-Value space into a series of consecutive Key segments. Each segment is called a Region. There is a size limit for each Region to store data (the default value is 96 MB and the size can be configured). Each Region can be described by `[StartKey, EndKey)`, a left-closed and right-open interval.
+TiKV chooses the second solution that divides the whole Key-Value space into a series of consecutive Key segments. Each segment is called a Region. When the size of a Region exceeds a threshold (144 MB by default), TiKV splits it to two or more Regions. There is a size limit for each Region to store data (the default value is 96 MB and the size can be configured). Each Region can be described by `[StartKey, EndKey)`, a left-closed and right-open interval.
```
Comment (hfxsd) [inline tidb-storage.md:52]:
```suggestion
TiKV chooses the second solution that divides the whole Key-Value space into a series of consecutive Key segments. Each segment is called a Region. When the size of a Region exceeds a threshold (144 MiB by default), TiKV splits it to two or more Regions. There is a size limit for each Region to store data (the default value is 96 MiB and the size can be configured). Each Region can be described by `[StartKey, EndKey)`, a left-closed and right-open interval.
```

File: tikv-overview.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ There is a RocksDB database within each Store and it stores data into the local
 
 Data consistency between replicas of a Region is guaranteed by the Raft Consensus Algorithm. Only the leader of the Region can provide the writing service, and only when the data is written to the majority of replicas of a Region, the write operation succeeds.
 
-When the size of a Region exceeds a threshold, which is 144 MB by default, TiKV splits it to two or more Regions. This operation guarantees the size of all the Regions in the cluster is nearly the same, which helps the PD component to balance Regions among nodes in a TiKV cluster. When the size of a Region is smaller than the threshold, TiKV merges the two smaller adjacent Regions into one Region.
+When the size of a Region exceeds a threshold (144 MB by default), TiKV splits it to two or more Regions. This operation guarantees the size of all the Regions in the cluster is nearly the same. The Region size is currently 96 MB by default in TiKV. This mechanism helps the PD component to balance Regions among nodes in a TiKV cluster. When the size of a Region is smaller than the threshold, TiKV merges the two smaller adjacent Regions into one Region.
```
Comment (hfxsd) [inline tikv-overview.md:24]:
```suggestion
When the size of a Region exceeds a threshold (144 MiB by default), TiKV splits it to two or more Regions. This operation guarantees the size of all the Regions in the cluster is nearly the same. The Region size is currently 96 MiB by default in TiKV. This mechanism helps the PD component to balance Regions among nodes in a TiKV cluster. When the size of a Region is smaller than the threshold, TiKV merges the two smaller adjacent Regions into one Region.
```

File: tidb-storage.md [hunk 3]
```diff
@@ -49,7 +49,7 @@ To make it easy to understand, let's assume that all data only has one replica.
 * Hash: Create Hash by Key and select the corresponding storage node according to the Hash value.
 * Range: Divide ranges by Key, where a segment of serial Key is stored on a node.
 
-TiKV chooses the second solution that divides the whole Key-Value space into a series of consecutive Key segments. Each segment is called a Region. There is a size limit for each Region to store data (the default value is 96 MB and the size can be configured). Each Region can be described by `[StartKey, EndKey)`, a left-closed and right-open interval.
+TiKV chooses the second solution that divides the whole Key-Value space into a series of consecutive Key segments. Each segment is called a Region. When the size of a Region exceeds a threshold (144 MiB by default), TiKV splits it to two or more Regions. There is a size limit for each Region to store data (the default value is 96 MiB and the size can be configured). Each Region can be described by `[StartKey, EndKey)`, a left-closed and right-open interval.
```
Comment (Oreoxmt) [inline tidb-storage.md:52]:
```suggestion
TiKV chooses the second solution that divides the whole Key-Value space into a series of consecutive Key segments. Each segment is called a Region. When the size of a Region exceeds a threshold (144 MiB by default), TiKV splits it into two or more Regions. There is a size limit for each Region to store data (the default value is 96 MiB and the size can be configured). Each Region can be described by `[StartKey, EndKey)`, a left-closed and right-open interval.
```

**Review by Oreoxmt — APPROVED** [review]

File: tikv-overview.md [hunk 4]
```diff
@@ -21,7 +21,7 @@ There is a RocksDB database within each Store and it stores data into the local
 
 Data consistency between replicas of a Region is guaranteed by the Raft Consensus Algorithm. Only the leader of the Region can provide the writing service, and only when the data is written to the majority of replicas of a Region, the write operation succeeds.
 
-When the size of a Region exceeds a threshold, which is 144 MB by default, TiKV splits it to two or more Regions. This operation guarantees the size of all the Regions in the cluster is nearly the same, which helps the PD component to balance Regions among nodes in a TiKV cluster. When the size of a Region is smaller than the threshold, TiKV merges the two smaller adjacent Regions into one Region.
+ TiKV tries to keep an appropriate size of each Region in the cluster. The Region size is currently 96 MiB by default. This mechanism helps the PD component to balance Regions among nodes in a TiKV cluster. When the size of a Region exceeds a threshold (144 MiB by default), TiKV splits it into two or more Regions. When the size of a Region is smaller than the threshold (20 MiB by default), TiKV merges the two smaller adjacent Regions into one Region.
```
Comment (Oreoxmt) [inline tikv-overview.md:24]:
```suggestion
TiKV tries to keep an appropriate size for each Region in the cluster. The Region size is currently 96 MiB by default. This mechanism helps the PD component to balance Regions among nodes in a TiKV node. When the size of a Region exceeds a threshold (144 MiB by default), TiKV splits it into two or more Regions. When the size of a Region is smaller than the threshold (20 MiB by default), TiKV merges the two smaller adjacent Regions into one Region.
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: tikv-overview.md (hunk 4 above)
Comment (pingyu) [reply tikv-overview.md:24]: I think it should be `among nodes in a TiKV cluster`.

**Review by pingyu — APPROVED** [review]
LGTM~

**Pull request merged**